packages/account/src/Components/poi/status/unsupported/unsupported.scss

```diff
@@ -1,26 +1,12 @@
 .manual-poi {
-    margin: 0 -1rem;
+    @include mobile {
+        margin: 3.2rem 1.6rem 1.6rem;
+    }
 
     &--mobile {
         margin: 0;
         padding: 4rem 0.6rem 0;
     }
```

Comment:
this classname is removed from the component. so we can safely remove these styles?